CHANGELOG.md

```diff
@@ -1,16 +1,20 @@
 
 # Changelog
 
-## [1.1.4 - Unreleased]
+## [1.1.5 - Unreleased]
 ### Changed
 ### Added
+- Log stacktrace with source filename, function and line number
+- Log relevant version information at startup of worker
```

Comment:
Approving, but these may not be necesassary?

Reply:
removed